### PR TITLE
Heat Stress Tracker Dashboard

### PR DESCRIPTION
**Pull request by avalmas-programize**
Create a new endpoint to serve the Heat Stress Tracker reef list
GET `/collections/heat-stress-tracker` fetches the heat stress tracker collection

Create a new type, `DynamicCollection`, which is a subset of the `Collection` model (minus id, user and updated/created columns)

### CODE REVIEW

**Review by Ry-DS — COMMENTED** [review]
Nice! Let me know what you think of the below comments :+1:

File: packages/api/src/collections/collections.service.ts [hunk 1]
```diff
@@ -174,4 +161,52 @@ export class CollectionsService {
...
+  async getHeatStressTracker() {
+    const heatStressData = await this.latestDataRepository.find({
+      metric: Metric.DHW,
+      value: MoreThan(0),
+    });
+
+    const heatStressReefIds = heatStressData.reduce<number[]>(
+      (reefIds, data) => [...reefIds, data.reefId],
+      [],
+    );
```
[the remark this answers is not in the document]
Comment (avalmas-programize) [reply packages/api/src/collections/collections.service.ts:174]: Yeah i don't know what I was thinking. Fixed

**Comment by ericboucher** [comment]
@tleraj let's add a section in the `/tracker` page that looks a bit similar to the first one but with this content:

Title - "Track global heat stress at a glance"
Text - Aqualink has developed a tracker that highlights all the sites under the most stress, at a glance. Use this page to see where the ocean is getting warmer and spot new heat waves.
Pic - Pic from the new tracker page

**Comment by tleraj** [comment]
@ericboucher Let me know what you think
Screenshot_20210719_144847

**Comment by ericboucher** [comment]
In case the reefs pan the whole globe, is it easy to pin the map to it's edges and avoid showing a grey area at the top or bottom? See below:

Screen Shot 2021-07-21 at 11 08 27 AM

**Review by ericboucher — APPROVED** [review]

File: packages/website/src/routes/Tracker/index.tsx [hunk 2]
```diff
@@ -22,6 +22,11 @@ import image1 from "../../assets/img/tracker-page/image1.png";
 import image2 from "../../assets/img/tracker-page/image2.png";
 import image3 from "../../assets/img/tracker-page/image3.png";
 
+const heroAspectRatio = 1054 / 300;
```
Comment (ericboucher) [inline packages/website/src/routes/Tracker/index.tsx:25]: Can't we get these programmatically from the image? It feels very odd to have this written in the code.

**Review by ericboucher — APPROVED** [review]

**Pull request merged**